core/src/main/java/discord4j/core/object/entity/User.java

```diff
@@ -223,6 +236,85 @@ public final int hashCode() {
         return EntityUtil.hashCode(this);
     }
 
+    /** Describes the flags of a user. */
+    public enum Flag {
```

Comment:
We need UNKNOWN value -1 for future compatibility in case more flags are added.

Check classes like MfaLevel, PremiumTier, etc

Reply:
After discussing it with Doc, this is fine - it's more like SystemChannelFlag. New flags won't cause errors, they are just going to be ignored.

Reply:
This system check every flag if discord add new flags this system only parse this like empty (if the user only have the new flag)